docs/env_vars.md

```diff
@@ -49,3 +49,4 @@ FLOAT16             | [1]        | use float16 for images instead of float32
 PTX                 | [1]        | enable the specialized [PTX](https://docs.nvidia.com/cuda/parallel-thread-execution/) assembler for Nvidia GPUs. If not set, defaults to generic CUDA codegen backend.
 PROFILE             | [1]        | enable output of [perfetto](https://ui.perfetto.dev/) compatible profile. This feature is supported in NV and AMD backends.
 VISIBLE_DEVICES     | [list[int]]| restricts the NV/AMD devices that are available. The format is a comma-separated list of identifiers (indexing starts with 0).
+JIT                 | [0-2]      | 0=disabled, 1=[jit enabled](/docs/quickstart.md#jit), 2=jit enabled but [splitting cache into batches is disabled for compatibility](https://github.com/tinygrad/tinygrad/issues/5408) |
```

Comment:
jit=2 is jit enabled, but graphs are disabled. it's not related to the linked issue or any compatibility

Reply:
understood! will rephrase